mxfusion/components/distributions/__init__.py

```diff
@@ -13,10 +13,11 @@
     random_gen
     univariate
     gp
+    laplace
```

Comment:
```suggestion
    laplace
    uniform
```

Reply:
This is now complete. I also added tests to the gamma and normal distributions to match.